i18n/english.yml

```
@@ -87,6 +87,9 @@ components:
         placeholder: Additional information (optional)
       publishNewVersion:
         label: Publish snapshot as new feed version
+      publishProprietaryFiles: 
+        helpText: Proprietary files allow you to maintain certain proprietary Datatools features across feed version, e.g. Pattern names.
+        label: Publish new feed version with proprietary (extra) datatools files.
```

```suggestion
        label: Publish new feed version with proprietary (extra) Datatools files.
```

Consistent capitalization

---

Good call, updated in https://github.com/ibi-group/datatools-ui/pull/991/commits/0f88ac57e5493d4bf7d7808169347b6d8ea9a788